cfme/cloud/keypairs.py

```diff
@@ -215,19 +226,19 @@ def exists(self):
         return view.is_displayed
 
 
-@navigator.register(KeyPair, 'All')
-class All(CFMENavigateStep):
+@navigator.register(Server)
+class CloudKeyPair(CFMENavigateStep):
```

Comment:
**Required**
This class name implies a single keypair, I'd like to see something like `CloudKeyPairs` or `KeyPairsCollection`, something along those lines that makes it more clear you're navigating to 'All'.

Additionally, consider keeping 'All' in the registration as the name, so that we can keep consistent navigation destinations, and I can say import KeyPair, Volume, and Instance, and navigate to 'All' against any of the classes.